src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptGlpyhHelpers.cs

```diff
@@ -0,0 +1,327 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript
+{
+    internal static class VSTypeScriptGlyphHelpers
+    {
+        public static Microsoft.CodeAnalysis.Glyph ConvertTo(VSTypeScriptGlyph glyph)
+        {
+            switch (glyph)
```

Comment:
Do we show "convert to switch expression" suggestion? If so, please make the change.

Reply:
Surprisingly, there was no suggestion, but I implemented the change anyways.